fail2ban/server/banmanager.py

```diff
@@ -26,6 +26,9 @@
 
 from threading import Lock
 
+import dns.exception
```

Comment:
What if a single "error" was returned for each? That could be more evident (munin graph with only "error" being printed), then check logs and see the module/import error